Install the libbson documentation files to a place that doesn't conflict
with the mongo-c-driver documentation files.
--- CMakeLists.txt.orig	2020-01-16 21:14:56.000000000 -0600
+++ CMakeLists.txt	2020-01-18 20:49:02.000000000 -0600
@@ -375,7 +375,7 @@
 endif ()
 
 install (FILES COPYING NEWS README.rst THIRD_PARTY_NOTICES
-   DESTINATION ${CMAKE_INSTALL_DATADIR}/mongo-c-driver
+   DESTINATION ${CMAKE_INSTALL_DATADIR}/libbson
 )
 
 if (ENABLE_UNINSTALL)
